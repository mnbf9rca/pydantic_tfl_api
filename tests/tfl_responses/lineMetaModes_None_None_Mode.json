{"status_code": 200, "headers": {"Date": "Mon, 15 Jul 2024 13:17:56 GMT", "Content-Type": "application/json; charset=utf-8", "Content-Length": "292", "Connection": "keep-alive", "Cache-Control": "public, must-revalidate, max-age=43200, s-maxage=86400", "Via": "1.1 varnish", "Content-Encoding": "gzip", "Accept-Ranges": "bytes", "Age": "86063", "Access-Control-Allow-Headers": "Content-Type", "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS", "Access-Control-Allow-Origin": "*", "Api-Entity-Payload": "LineMeta", "X-Backend": "api", "X-Cache": "HIT", "X-Cache-Hits": "2370", "X-Cacheable": "Yes. Cacheable", "X-Frame-Options": "deny", "X-Proxy-Connection": "unset", "X-TTL": "86400.000", "X-TTL-RULE": "0", "X-Varnish": "2024977757 2011801752", "X-AspNet-Version": "4.0.30319", "X-Operation": "Line_MetaModes", "X-API": "Line", "CF-Cache-Status": "DYNAMIC", "Strict-Transport-Security": "max-age=31536000", "Set-Cookie": "_cfuvid=d6iuqp.QgoYWhZbHl7YKbk.0wWmUzVZMa2BHNTqsYRA-1721049476493-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None", "Server": "cloudflare", "CF-RAY": "8a3a091babe063f0-LHR"}, "data": [{"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "bus"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "cable-car"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": true, "isScheduledService": true, "modeName": "coach"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": false, "isScheduledService": false, "modeName": "cycle"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": false, "modeName": "cycle-hire"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "dlr"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "elizabeth-line"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": false, "isScheduledService": false, "modeName": "interchange-keep-sitting"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": false, "isScheduledService": false, "modeName": "interchange-secure"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": true, "isScheduledService": true, "modeName": "national-rail"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "overground"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "replacement-bus"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "river-bus"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "river-tour"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": false, "isScheduledService": false, "modeName": "taxi"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "tram"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": true, "isFarePaying": true, "isScheduledService": true, "modeName": "tube"}, {"$type": "Tfl.Api.Presentation.Entities.Mode, Tfl.Api.Presentation.Entities", "isTflService": false, "isFarePaying": false, "isScheduledService": false, "modeName": "walking"}]}