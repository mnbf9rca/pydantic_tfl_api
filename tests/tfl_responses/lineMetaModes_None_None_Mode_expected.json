["{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"bus\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"cable-car\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"coach\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":false,\"is_scheduled_service\":false,\"mode_name\":\"cycle\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":false,\"mode_name\":\"cycle-hire\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"dlr\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"elizabeth-line\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":false,\"is_scheduled_service\":false,\"mode_name\":\"interchange-keep-sitting\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":false,\"is_scheduled_service\":false,\"mode_name\":\"interchange-secure\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"national-rail\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"overground\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"replacement-bus\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"river-bus\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"river-tour\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":false,\"is_scheduled_service\":false,\"mode_name\":\"taxi\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"tram\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":true,\"is_fare_paying\":true,\"is_scheduled_service\":true,\"mode_name\":\"tube\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}", "{\"is_tfl_service\":false,\"is_fare_paying\":false,\"is_scheduled_service\":false,\"mode_name\":\"walking\",\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}"]